### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by zqb-all**

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -331,11 +331,11 @@ msgstr "默认解释器"
 
 #: ../data/prefs.glade.h:49
 msgid "_Run command as a login shell"
-msgstr "运行命令作为登录外壳(_R)"
```
Comment (zqb-all) [inline po/zh_CN.po]: I don't know if the `(_R)` here is necessary

Comment (Stibbons) [inline po/zh_CN.po]: For the hotkey ?

Comment (zqb-all) [inline po/zh_CN.po]: Maybe, but I try to press "R" and nothing happen. Then I change my system language to English, 
 and the text there just `"Reset Compatibibility***"`, the R is just a normal text, and no tailing (_R), so I delete the (_R) in the translation.

**Comment by Stibbons** [comment]
Can I merge this PR or I let you repudiate the translations ?

**Comment by Stibbons** [comment]
Can I merge this PR or I let you reupdate the translations ?

**Comment by zqb-all** [comment]
I think this PR is ok to merge.
Can this closed PR be reopen to merge? or I build a new PR?

**Comment by Stibbons** [comment]
Of course it can !

**Pull request merged**